page-front.php

```diff
@@ -33,25 +33,17 @@
 
 			<?php while ( have_posts() ) : the_post(); ?>
 
-			<?php if ( is_active_sidebar( 'sidebar' ) ) { ?>
-
-				<div id="content" class="content has-sidebar">
+			<div id="content" class="content <?php if (is_active_sidebar( 'sidebar' )) { echo "has-sidebar";} ?>">
```

Comment:
This line is being flagged for incorrect spacing, and unneeded use of double quotes, in CodeSniffer:

```
FILE: ...ordpress/src/wp-content/themes/MITLibraries-child/page-front.php
----------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 1 LINE
----------------------------------------------------------------------
 36 | ERROR | [x] No space after opening parenthesis is prohibited
    |       |     (WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis)
 36 | ERROR | [x] No space before closing parenthesis is prohibited
    |       |     (WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBeforeCloseParenthesis)
 36 | ERROR | [x] String "has-sidebar" does not require double quotes;
    |       |     use single quotes instead
    |       |     (Squiz.Strings.DoubleQuoteUsage.NotRequired)
----------------------------------------------------------------------
PHPCBF CAN FIX THE 3 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```